### DISCUSSION TITLE
32 bit Compatibility, Lyrat Amp and posibly project-wiki

### DISCUSSION THREAD

**Discussion by AxelJust** [sle118/squeezelite-esp32#162] [General]
Hey,
I tested both Lyrat V4.3 and Olimex ESP32-ADF rev. B based on the ES8388
1036 32bit works on both.
Somehow with Identical settings `jack_mutes_amp` works as set on the Olimex. But on the Lyrat I don't get Audio from the Amp at all.
Has someone sujestions here?
I tried to find someone who the capasitive butons were setup in Squeezelite-esp32, but didn't get resuls.
Some tips or the code would be some great help.
I would like to suport the project. As I'm at the surface of Coding I would rather try to setup a wiki or Confihure a board library so that more SDK default files can get available.
Would that help?
What do you think?

**Comment by sle118** [comment]
Have you tried to look at the large dedicated thread in the Logitech forum? That would be a good source of information and a good place to ask the community.

https://forums.slimdevices.com/showthread.php?112697-ANNOUNCE-Squeezelite-ESP32-(dedicated-thread)

I don't have that board specifically so I can't help much, but when you find the right config, I could add it to the known boards (a new feature available in branch master 4.3)
